src/components/templates/Header.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from "react";
-import { Container, ListItem, Button } from "./index";
+import { Container, ListItem, Button } from "../index";
 import { RxHamburgerMenu } from "react-icons/rx";
 import { IoCloseCircleOutline } from "react-icons/io5";
 import { Link, NavLink } from "react-router-dom";
```

Comment:
Always sort your imports in a way that the local imports come after.

**global import**
import React from 'react';

**local imports**
import { Container } from './index';

**css if needed**
import './styles.css';